src/components/IIIFPlayerWrapper.js

```diff
@@ -4,26 +4,31 @@ import { useManifestDispatch } from '../context/manifest-context';
 import StructuredNavigation from '@Components/StructuredNavigation/StructuredNavigation';
 import PropTypes from 'prop-types';
 
-export default function IIIFPlayerWrapper({ manifestUrl, children }) {
-  const [manifest, setManifest] = useState();
+export default function IIIFPlayerWrapper({ manifestUrl, children, manifest }) {
+  const [manifestValue, setManifestValue] = useState(manifest);
```

Comment:
I'd like to keep this state identifier as `manifest`.  Let's convert the `manifest` prop identifier in line 7 to a new title?  `manifestValue` there instead?